pandas/core/base.py

```diff
@@ -1340,7 +1356,11 @@ def is_monotonic(self):
 
         return Index(self).is_monotonic
 
-    is_monotonic_increasing = is_monotonic
+    @property
+    def is_monotonic_increasing(self) -> bool:
+        """alias for is_monotonic"""
+        # mypy complains if we alias directly
```

Comment:
probably don't need the comment.

In Index, `is_monotonic` is an alias for `is_monotonic_increasing`, whereas here `is_monotonic_increasing` is an alias for `is_monotonic`. Would it make sense to make these consistent instead?

Although without going deeper, looks strange that `is_monotonic` returns `Index(self).is_monotonic`, maybe the code there, should be here.

Reply:
> maybe the code there, should be here.

it does seem a little weird, but its because this gets mixed in to Series, so Series.is_monotonic needs to wrap in Index so that it can use IndexEngine